### DISCUSSION TITLE
Finish code folding in one line

### DISCUSSION THREAD

**Discussion by Cyberavater** [zokugun/vscode-explicit-folding#58] [Q&A]
Can this extension solve the problem explained here?
https://stackoverflow.com/questions/68227011/finish-code-folding-in-one-line-and-keep-folding-sign-visible-all-the-time

If yes, please share the rule, I don't understand regex that much. Thanks!

**Comment by daiyam** [comment]
I replied directly on stack overflow.

**Reply by Cyberavater** [reply]
I've upvoted your answer (on stack overflow) but is it not possible to finish the folding in the same line as mentioned here? https://github.com/microsoft/vscode/issues/3352

**Reply by daiyam** [reply]
I've updated the answer

**Reply by Cyberavater** [reply]
Hmm, pressing enter still creates a new line in the folding, but maybe because it's indentation-based. but what about syntax-based? can you create rules for (..),{..},[..]
Eg code from dart(flutter):

```
OutlinedButton(
              onPressed: () {
                Navigator.pushNamed(
                  context,
                  SecondPage.id,
                );
              },
              child: Text("Page 2"),
            )
```
